protobuf/api/service.proto

```diff
@@ -8,6 +8,9 @@ package api;
 // right now we have conflicts with api.Service.
 service ServiceX {
   rpc Create (CreateServiceRequest) returns (CreateServiceResponse) {}
+
+  // Get returns a single Service specified in a request.
+  rpc Get(GetServiceRequest) returns (definition.Service) {}
```

Comment:
it's better to apply same pattern of storing return values in a wrapper. otherwise it'll not be consistent for all apis.

e.g.
```
message GetServiceResponse {
 definition.Service service = 1;
}
```

Reply:
Actually this is to be consistent with the new API.
Check new Execution grpc api ;)